docs/stat-boxes.md

```diff
@@ -0,0 +1,584 @@
+
+If you want to find the icon name for a particular icon, follow these steps:
+1. Find your spell or item on Wowhead. For example, [Enchant Ring - Binding of Haste](https://www.wowhead.com/item=128542/enchant-ring-binding-of-haste).
+2. Click the spell's/item's icon on Wowhead, next to the tooltip.
+3. A box will come up, showing the icon's name.
+
+![Get icon name from Wowhead](images/get-icon-name.jpg)
+
+For a custom icon, include an import of your icon image file in the top of the module, then reference it in a plain `<img src={icon_img_name} />`.
+- `icon={<img src={MasteryRadiusImage} style={{ border: 0 }} alt="Mastery effectiveness" />}` from the [Holy Paladin - Mastery Effectiveness](../src/Parser/Paladin/Holy/Modules/Features/MasteryEffectiveness.js) stat box.
+- note that MasteryRadiusImage is in `/public/img` not in the `Paladin/Holy` folder.
+  
+### Value
+
+To get a value into a percentage, use formatPercentage (import module). It's common practice to do calculations before and store with a variable/constant name, then reference the variable/constant name here.
+- ```value={`${formatPercentage(agonyUptime)} %`}```
+- ```value={`${((this.heals / totalCastsIncludingDp) || 0).toFixed(2)} players`}```
+- ```value={`${formatNumber(this.hps)} HPS`}```
+
+### Label
+
+For basic text with no tooltip, just use a basic double-quote string.
+- `label="Agony uptime"`
+
+For a basic tooltip via label: 
+```javascript
+label={(
+  <dfn data-tip={'This is the effective healing contributed by the Eveloping Mists buff.'}> // tooltip
+    Healing Contributed // label
+  </dfn>
+)}
+``` 
+
+For a longer tooltip via label --> see [Mistweaver Monk - Mana Tea](../src/Parser/Monk/Mistweaver/Modules/Talents/ManaTea.js) talent module stat box.
+
+![Mana Tea tooltip](images/mana-tea-box-tooltip.jpg)
+
+### Tooltip
+
+For a basic tooltip:
+```
+tooltip="Tooltip text here."
+```
+
+For longer tooltip variations, see [Holy Paladin - Beacon of Faith healing](../src/Parser/Paladin/Holy/Modules/PaladinCore/BeaconHealing.js) or [Holy Paladin - Holy Avenger](../src/Parser/Paladin/Holy/Modules/Talents/HolyAvenger.js) talent module stat boxes.
+
+![Beacon of Faith tooltip](images/beacon-faith-box-tooltip.jpg)
+![Holy Avenger tooltip](images/holy-avenger-box-tooltip.jpg)
+
+### Footer
+
+This footer bar is usually seen as part of the damage taken, healing done, or downtime module boxes. However, you can also adapt it to a display module -- see [the styled footer bar](#downtimes-styled-footer-bar) box variation section below.
+
+---
+
+## Multiple listing
+
+You can add multiple icons or text rows within a StatisticBox, if you want to show different parts of a module. These are built with a `<span>` element with smaller-styled icons within the `value={}`. 
+- **Horizontal**: Two or three icons could fit on one line before visually breaking the box. 
+- **Vertical**: This grouping works better visually if the display contains a lot of text.
+
+```javascript
+<span>
```

Comment:
The preferred method is to use the `Wrapper` component from `common/Wrapper`. This doesn't add any DOM elements which slightly improves performance (and since web performance is usually by a thousand small cuts, it's a lot of small things like this that makes it matter).